packages/fluentui/docs/src/examples/components/Input/Variations/index.tsx

```diff
@@ -54,6 +54,11 @@ const Variations = () => (
       description="An input can handle both input and wrapper slots for targetting the input and wrapper elements, respectively."
       examplePath="components/Input/Variations/InputExampleTargeting"
     />
+    <ComponentExample
+      title="Input with Labels"
+      description="An input can have a label"
+      examplePath="components/Input/Variations/InputExampleWithLabels"
```

Comment:
```suggestion
      examplePath="components/Input/Slots/InputExampleWithLabels"
```

I think that it should be under `Slots` section

Reply:
Created slots section.